tests/phpt/json/38_optimized_init_val.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
```suggestion
@ok
<?php
require_once 'kphp_tester_include.php';
```
It add some magic to enable this file for testing

Reply:
Thanks